ERCS/erc-8000.md

```diff
@@ -0,0 +1,513 @@
+---
+eip: 8000
```

Comment:
```suggestion
eip: 7635
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500.

Please also update the filename.